packages/components/src/font-size-picker/index.js

```diff
@@ -112,4 +119,4 @@ function FontSizePicker( {
 	);
 }
 
-export default withInstanceId( FontSizePicker );
```

Comment:
I guess this isn't related, but good catch removing the unneeded HOC 👍

Reply:
As noted in the PR description 🙂

Reply:
Oops, right. Sorry, my pre-weekend brain missed it 😉